Update Positions-WAVLab collaborator section (#242)
shinjiwlab/shinjiwlab.github.io: _pages/positions.md

- [ ] &nbsp; Please submit your application through [LTI](https://www.lti.cs.cmu.edu/apply-lti). Unfortunately, we may not respond to a direct email about the Ph.D. application.

### WAVLab Collaborator
- We provide training and research opportunities to students who are interested in speech processing and want to collaborate with us.

>   - We give priority to students at Carnegie Mellon University (CMU); students from other universities are also welcome. We accept both undergraduate and graduate students.

Reviewer: ```suggestion
  - We usually work with students at Carnegie Mellon University (CMU), but students from other universities are also welcome. We accept both undergraduate and graduate students if they are motivated.
```

>   - The opportunities are mainly educational-oriented, especially for students who are new to speech research. We guide students in understanding speech research background, required programming skills, knowledge of the cluster, and use of speech and audio toolkits. Students can also conduct research under our supervision. 

Reviewer: ```suggestion
  - We emphasize the educational perspectives for speech processing, especially for students who are new to speech research. We guide students in understanding speech research background, required programming skills, knowledge of the cluster, and use of speech and audio toolkits. Students can also conduct research under our supervision. 
```

  - It usually takes a few months or even longer for students to obtain sufficient research skills before starting serious research activities. We believe this process is necessary and is essential for new researchers. If you want to publish a paper as soon as possible, our lab is not the best option for you, unfortunately. We recommend you contact the other faculties.
- [ ] &nbsp; Students who are interested in these opportunities should fill out **[this form](https://docs.google.com/forms/d/1AE-MMtqXpdPG07U6nnNfp9N2abJrI54XeVIshkyrwng/)**
  - If you fail to do so, we regard that you do not thoroughly investigate our lab's activities on this webpage, and unfortunately, we may not respond to your application.

>   - The form will reqeest your CV. Note that your CV will be shared with other lab members (but we do not distribute it outside).

Reviewer 2: ```suggestion
  - The form will request your CV. Note that your CV will be shared with other lab members (but we do not distribute it outside).
```

### WAVLab Collaborator
- We provide training and research opportunities to students who are interested in speech processing and want to collaborate with us.
  - We give priority to students at Carnegie Mellon University (CMU); students from other universities are also welcome. We accept both undergraduate and graduate students.
  - The opportunities are mainly educational-oriented, especially for students who are new to speech research. We guide students in understanding speech research background, required programming skills, knowledge of the cluster, and use of speech and audio toolkits. Students can also conduct research under our supervision. 
  - It usually takes a few months or even longer for students to obtain sufficient research skills before starting serious research activities. We believe this process is necessary and is essential for new researchers. If you want to publish a paper as soon as possible, our lab is not the best option for you, unfortunately. We recommend you contact the other faculties.
- [ ] &nbsp; Students who are interested in these opportunities should: 

>   - Fill out **this form**

Reviewer 2: my understanding is that (please correct me if not true): there will be a form link attached here in later commits?

Reviewer 3: Created first draft of Google form here: https://docs.google.com/forms/d/1AE-MMtqXpdPG07U6nnNfp9N2abJrI54XeVIshkyrwng/
